apps/verilator/add/src/driver.cc

```diff
@@ -28,14 +28,14 @@ namespace runtime {
 namespace contrib {
 
 extern "C" VerilatorHandle VerilatorAlloc() {
-  Top *top = new Top;
+  Top* top = new Top;
```

Comment:
Do we currently have these sources running under a linter @vegaluisjose ?

Reply:
Not really, I forgot running the previous one with the clang-formatter